config/locales/models/user.fr.yml

```
@@ -48,6 +48,10 @@ fr:
               franceconnect_frozen_field_cannot_change: Le nom de naissance ne peut-être changé car il a été certifié par FranceConnect
             birth_date:
               franceconnect_frozen_field_cannot_change: La date de naissance ne peut-être changée car elle a été certifiée par FranceConnect
+    warnings:
+      models:
+        user:
+          ants_pre_demande_number_already_used: Le numéro de pré-demande ANTS renseigné, est déjà utilisé pour une prise de RDV auprès de %{meeting_point}. Veuillez dans un premier temps, annuler ce RDV en cliquant <a href='%{management_url}' target="_blank">ici<a>.
```

On peut augmenter les chances que l'usager lise cet avertissement avec un texte un peu plus succinct (et il faudra modifier la spec qui reprend ce texte).
En fonction de là où on place ce warning (si on le bouge dans la prise de rdv côté agent), on pourrait aussi ajouter explicitement le numéro de pré-demande (genre remplacer `Ce numéro de pré-demande ANTS est déjà` par `Le numéro de pré-demande ANTS ${ants_pre_demande_number} est déjà`

```suggestion
          ants_pre_demande_number_already_used: Ce numéro de pré-demande ANTS est déjà utilisé pour un RDV auprès de %{meeting_point}. Veuillez <a href='%{management_url}' target="_blank">annuler ce RDV<a> avant d'en prendre un nouveau.
```